etc/ci/update-wpt-checkout

```diff
@@ -67,12 +67,10 @@ function unsafe_run_tests() {
 # last commit with the new results.
 function unsafe_update_metadata() {
     ./mach update-wpt "${1}" || return 1
-    # Ignore any metadata changes to files that weren't modified by this sync.
-    git checkout -- tests/wpt/mozilla/meta || return 2
     # Ensure any new directories or ini files are included in these changes.
-    git add tests/wpt/metadata || return 3
+    git add tests/wpt/ || return 2
```

Comment:
Should this be `tests/wpt/metadata tests/wpt/mozilla/meta`, or do we actually want to include all the other files (some of which don't look like metadata)? If the latter, we may want to rename this function.